.luacov

```diff
@@ -20,12 +20,20 @@ return {
         -- (happens with integration tests)
         "^lib/",
         "^awesomerc.lua",
+
+        -- Include awesomerc.lua (in build dir with integration tests).
+        escape_pattern(build) .. "/awesomerc.lua",
```

Comment:
But `awesomerc.lua` is where the `luacov.runner` gets added, so that might be irrelevant..